src/main/webapp/app/core/course/manage/exercises-experimental/exercise-row/exercise-table.component.scss

```
@@ -0,0 +1,78 @@
+    }
+}
+
+// Compact card shown under the cursor while dragging a row (a detached <tr> would not render correctly).
+.exercise-drag-preview {
+    display: flex;
+    align-items: center;
+    padding: 0.4rem 0.75rem;
+    background-color: var(--bs-body-bg);
+    border: 1px solid var(--bs-border-color);
+    border-radius: var(--bs-border-radius);
+    box-shadow: 0 5px 12px rgb(0 0 0 / 20%);
```

Stylelint (17.13.0)

[error] Expected "20%" to be "0.2" (alpha-value-notation)

(alpha-value-notation)